bitmovin/resources/models/encodings/drms/cenc_playready_entry.py

```diff
@@ -0,0 +1,25 @@
+from bitmovin.errors import InvalidTypeError
+from bitmovin.resources.enums import ACLPermission
+from bitmovin.utils import Serializable
+from bitmovin.resources import AbstractIdResource
+
+
+class CENCPlayReadyEntry(Serializable):
+
+    def __init__(self, la_url=None, pssh=None):
+        super().__init__()
+        self.laUrl = la_url
+        self.pssh = pssh
+
+    @classmethod
+    def parse_from_json_object(cls, json_object):
+        pssh = json_object['pssh']
+        la_url = json_object('laUrl')
```

Comment:
i think that this (`json_object('laUrl')`) will not work. please use json_object.get('laUrl') here (and probably in the line above too).

when you call json_object['abc'] and abc does not exist you'll get an `AttributeError`. when fetching the value using json_object.get('abc') you will get `None` if abc doesn't exist.